docs/src/example.md

```diff
@@ -67,11 +67,28 @@ julia> MOI.get(optimizer, MOI.PrimalStatus())
 Given the optimized solution, we check that its value is, as expected, equal to `28/3`, and the solution vector `x` is `[4/3, 4/3]`:
 
 ```julia
-julia> isapprox.(MOI.get(optimizer, MOI.ObjectiveValue()), 28/3, atol = 1e-4)
-julia> isapprox.(MOI.get(optimizer, MOI.VariablePrimal(), x[1]), 4/3, atol = 1e-4)
-julia> isapprox.(MOI.get(optimizer, MOI.VariablePrimal(), x[2]), 4/3, atol = 1e-4)
+julia> isapprox(MOI.get(optimizer, MOI.ObjectiveValue()), 28/3, atol = 1e-4)
+julia> isapprox(MOI.get(optimizer, MOI.VariablePrimal(), x[1]), 4/3, atol = 1e-4)
+julia> isapprox(MOI.get(optimizer, MOI.VariablePrimal(), x[2]), 4/3, atol = 1e-4)
+```
+
+We can also retrieve the dual values associated to each parameter, **as they are all additive**:
+
+```julia
+julia> MOI.get(optimizer, MOI.ConstraintDual(), cy)
+julia> MOI.get(optimizer, MOI.ConstraintDual(), cz)
+julia> MOI.get(optimizer, MOI.ConstraintDual(), cw)
+```
+
+Notice the direct relationship in this case between the parameters' duals and the associated constraints' duals.
+The  `y` parameter, for example, only appears in the `cons1`. If we compare their duals, we can check that the dual of `y` is equal to its coefficient in `cons1` multiplied by the constraint's dual itself, as expected:
```

Comment:
there is one tricky point which is the sign of the dual.
Since parameters are like constants, we can think them as members of the right hand side (RHS) of a constraint.
However, the RHS thing does not generalize well to conic, hence, thinking of parameters as members of the LHS might be indeed better.
I think we might need a sign switch and the proper discussion in the docs.